### PR TITLE
When creating a new level, it should inherit the organization from WorkflowLevel1

### PR DESCRIPTION
**Pull request by Menda**
## Ticket

https://github.com/toladata/TolaDataV2/issues/366

### CODE REVIEW

File: indicators/test.py [hunk 1]
```diff
@@ -24,32 +24,32 @@ def setUp(self):
...
-        #Check for Indicator object
         get_indicator = Indicator.objects.get(name="testindicator")
         self.assertEqual(Indicator.objects.filter(id=get_indicator.id).count(), 1)
 
     def test_collected_exists(self):
-        #Check for CollectedData object
         get_collected = CollectedData.objects.get(description="somevaluecollected")
-        self.assertEqual(CollectedData.objects.filter(id=get_collected.id).count(), 1)
\ No newline at end of file
+        self.assertEqual(CollectedData.objects.filter(id=get_collected.id).count(), 1)
```
Comment (jefmoura) [inline indicators/test.py]: Can replace the two lines with one:
`self.assertEqual(len(CollectedData.objects.filter(description="somevaluecollected")), 1)`

File: indicators/test_models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from indicators.models import Level
+from workflow.models import WorkflowLevel1, Organization
+
+
+class LevelModelTest(TestCase):
+    def setUp(self):
```
Comment (jefmoura) [inline indicators/test_models.py]: Do you think we need setUp here? You could do everything inside of the test.

**Comment by Menda** [comment]
@jefmoura Let's do the refactoring of `indicators/test.py` in other ticket better.

**Comment by Menda** [comment]
@jefmoura done the change you asked in my test

**Pull request merged**